### PR TITLE
Added rowHighlightedId

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Required for support highlighting rows for flyout iterators.

### CODE REVIEW

File: src/app/visual/list-view-grid/list-view-grid-visual.component.html [hunk 1]
```diff
@@ -7,7 +7,10 @@
       </sky-list-column-selector-action>
     </sky-list-toolbar>
 
-    <sky-list-view-grid fit="scroll" #grid>
+    <sky-list-view-grid
+      fit="scroll"
+      #grid
```
Comment (Blackbaud-SteveBrush) [inline src/app/visual/list-view-grid/list-view-grid-visual.component.html:13]: Think we could add a visual test to demonstrate the highlighted style, just to make sure its getting translated through?

Comment (Blackbaud-AlexKingman) [reply src/app/visual/list-view-grid/list-view-grid-visual.component.html:13]: done

**Comment by blackbaud-johnly** [comment]
Added https://github.com/blackbaud/skyux2/issues/2248 to document new property.

File: src/app/visual/list-view-grid/list-view-grid-visual.component.ts [hunk 2]
```diff
@@ -21,4 +22,8 @@ export class ListViewGridTestComponent {
     { id: '6', column1: 606, column2: 'Lemon', column3: 'Larry eats lemons' },
     { id: '7', column1: 707, column2: 'Strawberry', column3: 'Sally eats strawberries' }
   ]);
+
+  public onToggleRowHighlightClick() {
+    this.rowHighlightedId = '2';
```
Comment (Blackbaud-SteveBrush) [inline src/app/visual/list-view-grid/list-view-grid-visual.component.ts]: The method doesn't actually toggle the highlight. What about something like?

```suggestion
    this.rowHighlightedId = (this.rowHighlightedId) ? undefined : '2';
```

Comment (Blackbaud-AlexKingman) [reply src/app/visual/list-view-grid/list-view-grid-visual.component.ts]: good call. done.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**